src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,149 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { getTodo } from '../../api/api';
+import { Todo } from '../../react-app-env';
+
 import './TodoList.scss';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
+type Props = {
+  onUserIdChange: (userId: number) => void,
+};
+
+export const TodoList: React.FC<Props> = ({ onUserIdChange }) => {
+  const [todos, setTodos] = useState<Todo[] | []>([]);
```

Comment:
suggestion: if you add `...| []` just to make it a default value, you'd rather implement it the following way
```suggestion
  const [todos, setTodos] = useState<Todo[] | null>(null);
```